eng/Packages.Data.props

```diff
@@ -92,7 +92,7 @@
     <PackageReference Update="System.Memory.Data" Version="6.0.0" />
     <PackageReference Update="System.Numerics.Vectors" Version="4.5.0" />
     <PackageReference Update="System.Net.Http" Version="4.3.4" />
-    <PackageReference Update="System.Diagnostics.DiagnosticSource" Version="6.0.1" />
+    <PackageReference Update="System.Diagnostics.DiagnosticSource" Version="8.0.0" />
```

Comment:
This one needs to be bumped because of Microsoft.Extensions.Logging.Abstractions